src/api/controllers/business-controllers.ts

```diff
@@ -0,0 +1,86 @@
+import { businessServices } from '../services';
+import { Request, Response, NextFunction } from 'express';
+
+const getBusinessById = async (req: Request, res: Response, next: NextFunction) => {
+    try {
+        /** Call to service layer */
+        const businessId = req.params.businessId.toString();
+        const result = await businessServices.getBusinessById(businessId);
+
+        /** Return a response to client */
+        return res.status(200).json(result);
+    } catch (err) {
+        next(err);
+    }
+};
+
+const getEmployeeList = async (req: Request, res: Response, next: NextFunction) => {
+    try {
+        /** Call to service layer */
+        const businessId = req.params.businessId.toString();
+        const result = await businessServices.getEmployeeList(businessId);
+
+        /** Return a response to client */
+        return res.status(200).json(result);
+    } catch (err) {
+        next(err);
+    }
+};
+
+const createSimpleBusiness = async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
createBusiness instead of simple business?

Reply:
createSimpleBusiness is a test-only route